packages/v4/patternfly-docs/content/accessibility/modal/modal.md

```diff
@@ -3,28 +3,94 @@ id: Modal
 section: components
 ---
 
-A **modal** displays important information to a user without requiring them to navigate to a new page.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+import { Link } from '@patternfly/documentation-framework/components/link/link';
 
-**Keyboard and mouse users** shouldn’t be able to interact with the rest of the page outside of an open modal. Set initial focus on the first focusable element in the modal. The user should be able to Tab to any interactive elements within the modal and use Tab + Shift to move backward through interactive elements. Keyboard users should be able to leave the modal by pressing Esc and focus should return to the element that invoked the modal. 
+## Accessibility
 
-**Screen reader users** shouldn’t be able to interact with the rest of the page outside an open modal. Add alternative text for any images or non-textual buttons (like icon buttons). Images should use the alt attribute: `aria-label` is most common to label icon buttons. Write a clear headline that describes the modal, and be sure to follow our modal content guidelines to clearly communicate information contained within it. 
+To implement an accessible PatternFly **modal** component:
 
-Bear in mind that any components placed inside the modal should also follow that component’s accessibility requirements.
-<br/>
-<br/>
+- Add an `aria-label` to the modal if it doesn't have a visible text title
+- Provide at least one clickable button that can close the modal, such as the default close icon or a "cancel" button
+- Ensure any other PatternFly components follow their respective accessibility documentation
 
-**To make modal accessible:**
+## Testing
 
-- **If you don’t use a title in your modal**, add an `aria-label` to give the modal an accessible name. It will look like `aria-label=”[title of modal]” on the AboutModal and will appear directly on the class .pf-c-about-modal-box. 
-- Use `aria-describedby` for any description you add to the modal.
+At a minimum, a modal should meet the following criteria:
 
-<br/>
-The following props can be added or are customizable in PatternFly:
+<List isPlain>
+  <ListItem>
+    <Checkbox id="modal-a11y-checkbox-1" label="Any content that is on the page, but outside of the modal, cannot be interacted with or navigated to while the modal is open." description={<span>Learn more about <Link href="/accessibility/product-development-guide/#trapping-focus">trapping focus</Link> in our product development guide for accessibility.</span>} />
```

Comment:
This is kind of picky, but I would link the words "product development guide" instead of trap focus since that's what you're actually linking to! Otherwise this looks great, and the guidelines are super clear!!